### ISSUE TITLE
Testnet state diff at 1042537

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#2337]
```
2022-01-24T22:31:57.524+0300    INFO    persisted to disk       {"blocks": 1, "keys": 23, "headerHeight": 1042537, "blockHeight": 1042537, "took": "3.626826ms"}
```
Version 0.98.1-pre-95-g42769d11.

**Comment by roman-khimov** [comment]
```
at 975644: 15ff7e2efe980bf370731e236860eb3ec5c4c42d738d200fb29a83f7427e8e58 vs 7ce0646cd4a9f7a85176f86f22b913d94e148f8b91d43698def9cb81619933da
at 975640: 9ebf5f60694f7ab36a0da7f8b3169d00e7dc981b5446240d38cbb5d2444c9355 vs 9ebf5f60694f7ab36a0da7f8b3169d00e7dc981b5446240d38cbb5d2444c9355
at 975642: fce6684ce0ec1dd86d51f11c785db65ad4f45e5e8a192f9abd3d831bacbe0b08 vs fce6684ce0ec1dd86d51f11c785db65ad4f45e5e8a192f9abd3d831bacbe0b08
at 975643: dbd493a39b675d5d754f80093ded218f10132bc6ac4fd91163b382f2227cb53d vs dbd493a39b675d5d754f80093ded218f10132bc6ac4fd91163b382f2227cb53d
state differs at 975644, block 968498b605351e01e266f8cabc4dbdebff0a4ffaec2862fa3088f30b424536bb
transaction a53925381d2b0e38a6025c0a3a05076c4ccb57c46021fd47ee0a812f8c9da424: diff
--- http://seed2t4.neo.org:20332
+++ http://localhost:20332
@@ -1,2 +1,2 @@
-(*result.ApplicationLog)(0xc0002a1980)({
+(*result.ApplicationLog)(0xc00012e1c0)({
  Container: (util.Uint256) (len=32 cap=32) 24a49d8c2f810aee47fd2160c457cb4c6c07053a0a5c02a6380e2b1d382539a5,
@@ -6,9 +6,7 @@
    Trigger: (trigger.Type) Application,
-   VMState: (vm.State) HALT,
-   GasConsumed: (int64) 6440674,
-   Stack: ([]stackitem.Item) (len=2 cap=2) {
-    (stackitem.Null) Null,
-    (stackitem.Null) Null
+   VMState: (vm.State) FAULT,
+   GasConsumed: (int64) 3410217,
+   Stack: ([]stackitem.Item) {
    },
-   Events: ([]state.NotificationEvent) (len=2 cap=4) {
+   Events: ([]state.NotificationEvent) (len=1 cap=4) {
     (state.NotificationEvent) {
@@ -16,11 +14,6 @@
      Name: (string) (len=8) "Transfer",
-     Item: (*stackitem.Array)(0xc0001341a0)(Array)
-    },
-    (state.NotificationEvent) {
-     ScriptHash: (util.Uint160) (len=20 cap=20) a39b664c3fadde872b1e3b3b4518aadf7d54b69c,
-     Name: (string) (len=8) "Transfer",
-     Item: (*stackitem.Array)(0xc000134200)(Array)
+     Item: (*stackitem.Array)(0xc00029c2a0)(Array)
     }
    },
-   FaultException: (string) ""
+   FaultException: (string) (len=60) "at instruction 22 (SYSCALL): expected byte size of 20 got 42"
   }
```

**Comment by roman-khimov** [comment]
It's fixed in neo-project/neo#2654 and there are no other known differences up to 1077876 testnet and 998060 mainnet.
